src/Http/Headers/ref/Microsoft.Net.Http.Headers.netcoreapp3.0.cs

```diff
@@ -118,74 +118,87 @@ public partial class EntityTagHeaderValue
     }
     public static partial class HeaderNames
     {
-        public const string Accept = "Accept";
```

Comment:
I think this is a breaking change? These can't be used in attributes anymore?

Reply:
Indeed. Do we have any examples where it's relevant?

Reply:
Was one in http2 tests; though that was testing the headers

![image](https://user-images.githubusercontent.com/1142958/56101494-21a9a600-5f1c-11e9-9c07-e8139a28d9ef.png)

Wouldn't think it was common?

Reply:
Suppose its a choice:

* Fast header string matching (extremely common; used in every request, 1258 uses in ASP.NET Core itself)

* Use in attributes (extremely uncommon, 2 test uses in ASP.NET Core itself and those testing http/2 protocol psudeo headers)

Reply:
It’s not common no. The only other thing is that I’d like to eventually burn the headers assembly to the ground

Reply:
Furthermore, I don't think this would be a binary breaking change since anything that referenced the constant would still have the constant burned into the assembly without ever actually referencing the HeaderNames class in the IL.

Reply:
@halter73 is correct. We should announce it as breaking but I don't have a problem taking it.

Reply:
I did a quick search and it looks like this API is used at least a little bit by "the Internet". Should we keep it around to avoid the breaking change?

(I agree it's only a source code breaking change, but that's still a breaking change.)

Reply:
Keeping it would involve duplication, which would get pretty confusing.

Reply:
To caveat the scope, its source code breaking if used in attributes or other string const initalization; otherwise isn't source code breaking.

Reply:
Oh I didn't see the whole change. I think this change is fine then. The few cases I found on the internet were not in attribute cases where a const is required. They were just regular imperative code. Carry on.

Reply:
We use these quite a bit in `case` statements for what it's worth...though I guess not anymore. Bleh.

Reply:
Sorry :-/